include/SDL3/SDL_events.h

```diff
@@ -559,6 +562,18 @@ typedef struct SDL_SensorEvent
     Uint64 sensor_timestamp; /**< The timestamp of the sensor reading in nanoseconds, not necessarily synchronized with the system clock */
 } SDL_SensorEvent;
 
+/**
+ *  \brief Location event structure (event.location.*)
+ */
+typedef struct SDL_LocationEvent
+{
+    Uint32 type;        /**< ::SDL_EVENT_LOCATION */
+    Uint64 timestamp;   /**< In nanoseconds, populated using SDL_GetTicksNS() */
+    double latitude;    /**< Latitude in degrees */
+    double longitude;   /**< Longitude in degrees */
+    double altitude;    /**< Altitude in meters */
```

Comment:
You should add "accuracy". Most apps ignore inaccurate position or display a circle to show accuracy to user.